_includes/current_sponsors.html

```diff
@@ -1,11 +0,0 @@
-<div id="current-sponsors-container">
```

Comment:
Did this entire file get deleted? Because that could cause some issues with the changes on your other branch if so.